PORT-8039 | PORT-8046 | Humanitec Script Integration (#1247)
port-labs/port-docs: docs/actions-and-automations/setup-backend/github-workflow/examples/Humanitec/create-humanitec-application.md

          "icon": "Environment"
        },
        "environment_name": {
          "type": "string",
          "description": "The Human-friendly name for the Environment.",
          "title": "Environment Name",
          "icon": "Environment"
        }
      },
      "required": [
        "application_name"
      ],
      "order": [
        "application_name",
        "environment",
        "environment_name",
        "environment_type"
      ]
    },
    "blueprintIdentifier": "humanitecApplication"
  },
  "invocationMethod": {
    "type": "GITHUB",
    "org": "mk-armah",
    "repo": "jira-actions",
    "workflow": "create-humanitec-application.yaml",
    "workflowInputs": {
      "application_name": "{{ .inputs.\"application_name\" }}",
      "environment": "{{ .inputs.\"environment\" }}",
      "environment_type": "{{ .inputs.\"environment_type\" }}",
      "environment_name": "{{ .inputs.\"environment_name\" }}"
    },
    "reportWorkflowStatus": true
  },
  "requiredApproval": false
}
```
</details>

Now you should see the `Create Application` action in the self-service page. 🎉

## Let's test it!

> 1. Head to the [Self Service hub](https://app.getport.io/self-serve)

Reviewer: ```suggestion
1. Go to the [Self Service page](https://app.getport.io/self-serve) of your portal.

```

Reviewer: @mk-armah  Please fix this in all "Let's test it" sections

> 2. Click on the `Create Application` action

Reviewer: ```suggestion
2. Click on the `Create Application` action.

```

3. Enter the required details for the `Application Name`, and optionally include `Environment`, `Environment Type`, and `Environment Name`.

> 4. Click on `Execute`

Reviewer: ```suggestion
4. Click on `Execute`.

```

> 5. Done! wait for the application to be created in Humanitec

Reviewer: ```suggestion
5. Done! wait for the application to be created in Humanitec.

```
